.github/CODEOWNERS

```diff
@@ -0,0 +1,53 @@
+# CODEOWNERS info & syntax
+# Lines starting with "#" are comments.
+#
+# Each line is a file pattern followed by one or more owners.
+#
+# Order is important; the last matching pattern takes the most
+# precedence.
+#
+# Owners can be specified by email address or GitHub username
+#
+# Teams can be specified as code owners as well. Teams should
+# be identified in the format @org/team-name. Teams must have
+# explicit write access to the repository.
+#
+# Patterns
+#
+# Whole repository
+# * @global-owner
+#
+# Directory (without subdirectories)
+# docs/* @tech_writer
+#
+# Directory (including subdirectories)
+# apps/ @app_developer
+#
+# Adding a leading "/" to a pattern means the directory must
+# be in the root of the repository.
+#
+# Empty Pattern -> no owner (@app_developer owns all of apps/ except apps/github)
+# apps/ @app_developer
+# apps/github
+
+# Uyuni Code Owners
+
+# Release Engineering
+rel-eng/ @release-engineering
+tito.props @release-engineering
```

Comment:
I seem to remember in some cases developers need to adjust this?

@mcalmer?

Reply:
Actually I'm not sure if it works anyway with a filename. Others can still change `tito.props`, but with this line we make sure that you know about the change (automatic pull request review).

Reply:
yes, but this is for reviews. So when somebody change it, ping the release-engineers